src/compile/common.ts

```diff
@@ -110,7 +136,7 @@ export function formatSignalRef(fieldDef: FieldDef<string>, specifiedFormat: str
  * @param format explicitly specified format
  */
 export function numberFormat(fieldDef: FieldDef<string>, specifiedFormat: string, config: Config) {
-  if (fieldDef.type === QUANTITATIVE) {
+  if (fieldDef.type === QUANTITATIVE || fieldDef.type === ORDINAL) {
```

Comment:
That was what I'd done once. But it added `format: 's'` to a lot of generated specs